source/jormungandr/jormungandr/default_settings.py

```diff
@@ -24,6 +24,11 @@
 
 DISABLE_DATABASE = boolean(os.getenv('JORMUNGANDR_DISABLE_DATABASE', False))
 
+# Active the asynchronous ridesharing mode
+ASYNCHRONOUS_RIDESHARING = boolean(os.getenv('JORMUNGANDR_ASYNCHRONOUS_RIDESHARING', False))
+# Active ridesharing service call with async greenlet
+GREENLET_POOL_FOR_RIDESHARING_SERVICES = boolean(os.getenv('JORMUNGANDR_GREENLET_POOL_FOR_RIDESHARING', False))
```

Comment:
Can we also have this in the configuration DB ?

Reply:
yes we need to have it, to change the value by the fly. In case of problems

Reply:
:+1:

Reply:
I think you also need

`GREENLET_POOL_SIZE_FOR_RIDESHARING_SERVICES`